pubsublite/streaming-analytics/src/test/java/PubsubliteToGcsIT.java

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.apache.beam.examples.common.WriteOneFilePerWindow;
+import org.apache.beam.sdk.coders.StringUtf8Coder;
+import org.apache.beam.sdk.testing.PAssert;
+import org.apache.beam.sdk.testing.TestPipeline;
+import org.apache.beam.sdk.testing.TestStream;
+import org.apache.beam.sdk.transforms.windowing.AfterProcessingTime;
+import org.apache.beam.sdk.transforms.windowing.FixedWindows;
+import org.apache.beam.sdk.transforms.windowing.IntervalWindow;
+import org.apache.beam.sdk.transforms.windowing.Repeatedly;
+import org.apache.beam.sdk.transforms.windowing.Window;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.TimestampedValue;
+import org.joda.time.Duration;
+import org.joda.time.Instant;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class PubsubliteToGcsIT {
+  @Rule public final TestPipeline testPipeline = TestPipeline.create();
+
+  private Instant baseTime = new Instant(0);
+
+  @Test
+  public void testPubsubliteToGcs() {
+    // Create a test stream.
```

Comment:
The tests should run the sample with GCP infrastructure and look at GCS to confirm it worked rather than testing the Beam SDK. I do understand the amount of work for that, so I am ok accepting this with a bug to update furhter.

Reply:
@averikitsch Do you know an example that I can look for how to write that kind of test? I just created https://github.com/GoogleCloudPlatform/java-docs-samples/issues/5113 for tracking. Thanks for the review!